src/changelog.md

```diff
@@ -1,6 +1,12 @@
 (changelog)=
 # Changelog
 
+(changelog-202311224)=
+## 2023-12-24
```

Comment:
Will this be a Christmas present? :upside_down_face: :christmas_tree: 

Or do we prefer to release it today?
```suggestion
(changelog-20231214)=
## 2023-12-14
```